st3/lsp_utils/generic_client_handler.py

```diff
@@ -112,16 +112,6 @@ def read_settings(cls) -> Tuple[sublime.Settings, str]:
         changed = cls.on_settings_read(loaded_settings)
```

Comment:
`cls.on_settings_read_internal(loaded_settings)` also mutates the `loaded_settings`. 
It sets the `languages` key.

```
    def on_settings_read_internal(cls, settings: sublime.Settings) -> None:
        languages = settings.get('languages', None)  # type: Optional[List[LanguagesDict]]
        if languages:
            settings.set('languages', cls._upgrade_languages_list(languages))
```

Reply:
Removed that too.

I thought that the LSP would complain without it but it seems like it has all the logic needed to handle old-style `languages`. Maybe that was added after being implemented in lsp_utils...